crmsh/bootstrap.py

```diff
@@ -2159,105 +2160,42 @@ def start_qdevice_on_join_node(seed_host):
     status_done()
 
 
-def remove_ssh():
-    seed_host = _context.cluster_node
-    if not seed_host:
-        error("No existing IP/hostname specified (use -c option)")
-
-    # check whether corosync has been started on local node
-    if not utils.service_is_active("corosync.service"):
-        error("Cluster is not active - can't execute removing action")
-
-    remove_get_hostname(seed_host)
-
-
-def remove_get_hostname(seed_host):
+def get_cluster_node_ip():
```

Comment:
This function name (`get_cluster_node_ip`) is pretty strange. You mention `get`, but the method is not used to get anything. It actually sets `_context.cluster_node_ip` value. We should change the method name to avoid confusions. Maybe `set_cluster_node_ip` would be better (just the fist thing coming to my mind)

Reply:
OK, Changed this:)